### PR TITLE
Set Correct Type for Boolean Attributes

### PR DESCRIPTION
**Pull request by Shaaali**
**Brief Description**
The Attribute type of boolean attributes such as `Active` are not correctly set in the scim2/Schemas API request. 
This PR fixes this issue (https://github.com/wso2/product-is/issues/13978)

**Related issues**
https://github.com/wso2/product-is/issues/13978

### CODE REVIEW

**Comment by Shaaali** [comment]
PLEASE DON'T MERGE THIS PR UNTIL THE FRONT END FIX FOR THIS IS MERGED.

**Comment by Shaaali** [comment]
Please merge https://github.com/wso2-enterprise/asgardeo-app-extensions/pull/1818 BEFORE merging this PR

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java
Comment (senthalan) [inline components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java:5576]: why were are hardcoding the active name here ?
we should check this from the scim2 configuration file and decide whether it is a bool right ?

Comment (Shaaali) [reply components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java:5576]: We have used a similar hardcoded approach for all the methods such as isComplex, isMultiValued etc. here : https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/blob/87d9c7763f64da2ce22ce3689525a32c58ca5e46/components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java#L5554

Therefore, added this method following the same approach.

**Review by senthalan — APPROVED** [review]

**Review by ShanChathusanda93 — APPROVED** [review]

**Pull request merged**
